arbor/util/rangeutil.hpp

```diff
@@ -9,6 +9,7 @@
 #include <iterator>
 #include <ostream>
 #include <numeric>
+#include <cstring>
```

Comment:
Are you sure? I don't see anything using this.

Reply:
std::memset?

Reply:
:eyes: